modules/gapi/CMakeLists.txt

```diff
@@ -320,20 +320,17 @@ if(HAVE_GAPI_ONEVPL)
   endif()
 
   if(UNIX)
-    find_package(PkgConfig)
-    if(PkgConfig_FOUND)
-      pkg_check_modules(PKG_LIBVA libva>=1.2 libva-drm>=1.2)
-      if(PKG_LIBVA_FOUND)
-        set(CMAKE_THREAD_PREFER_PTHREAD TRUE)
-        set(THREADS_PREFER_PTHREAD_FLAG TRUE)
-        find_package(Threads REQUIRED)
-      else()
-        message(FATAL_ERROR "libva not found: building HAVE_GAPI_ONEVPL without libVA support is impossible on UNIX systems")
+    if(WITH_VA)
+      include("${OpenCV_SOURCE_DIR}/cmake/OpenCVFindVA.cmake")
+      if(VA_INCLUDE_DIR)
+        ocv_target_include_directories(${the_module} SYSTEM PRIVATE ${VA_INCLUDE_DIR})
+        ocv_target_include_directories(opencv_test_gapi SYSTEM PRIVATE ${VA_INCLUDE_DIR})
+        ocv_target_link_libraries(${the_module} PRIVATE ${VA_LIBRARIES})
+        ocv_target_link_libraries(opencv_test_gapi PRIVATE ${VA_LIBRARIES})
       endif()
-    else()
-      message(FATAL_ERROR "PkgConfig not found: building HAVE_GAPI_ONEVPL without libVA support is impossible on UNIX systems")
-    endif()
-    ocv_target_link_libraries(${the_module} PRIVATE ${PKG_LIBVA_LIBRARIES} ${PKG_THREAD_LIBRARIES})
+    else(WITH_VA)
+      message(FATAL_ERROR "libva not found: building HAVE_GAPI_ONEVPL without libVA support is impossible on UNIX systems")
```

Comment:
Is this really a fatal error? Shouldn't the functionality be disabled instead?

Reply:
good question.
VPL on linux doesn't work on CPU. So without VA_API it is useless
What do you think - should we allow to configure VPL without VAAPI for now?

Reply:
ok, will see